receiver/awscontainerinsightreceiver/internal/gpu/dcgmscraper.go

```diff
@@ -25,7 +26,7 @@ import (
 )
 
 const (
-	caFile                    = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	caFile                    = "/etc/amazon-cloudwatch-observability-appsignals-cert/tls-ca.crt"
```

Comment:
Is this not used anywhere else when talking to the api server or kubelet or anything?

Reply:
This should be configurable ideally.

Reply:
appsignals seems misleading. Is this the same cert used for our webhook? or is it new?